### PR TITLE
Feature 122/create invitation

### PR DESCRIPTION
**Pull request by Maria358**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/CreateInvitationForm.vue [hunk 1]
```diff
@@ -225,13 +205,20 @@ export default defineComponent({
     onChangeApartment(apartmentId: number): void {
       this.apartmentId = apartmentId;
     },
+    correctType(): string {
+      const statusMap: any = {
+        ОСББ: 'cooperation',
+        Квартира: 'apartment',
```
Comment (aleonovdementiev) [inline src/components/CreateInvitationForm.vue]: It's better to use only english keys in objects, but you can set values on multiple languages

File: src/store/invitations/actions.ts [hunk 2]
```diff
@@ -16,16 +16,16 @@ import { AxiosResponse } from 'axios';
 export const actions: ActionTree<InvitationsStateInterface, RootStateInterface> & Actions = {
   [InvitationsActionsEnum.CREATE_INVITATION]: async ({ commit }, payload) => {
     try {
-      const payloadToSend = new PostInvitationDTOModel(payload);
+      const payloadToSend = new PostInvitationDTOModel(payload.payload);
```
Comment (aleonovdementiev) [inline src/store/invitations/actions.ts]: Maybe, it would be better to name variable 'data' instead of 'payload' for easier reading. F.e.: 'data.payload'

Comment (Maria358) [reply src/store/invitations/actions.ts]: Yes, I'll correct it

File: src/components/CreateInvitationForm.vue (hunk 1 above)
Comment (Maria358) [reply src/components/CreateInvitationForm.vue]: Updated))

**Review by aleonovdementiev — APPROVED** [review]

**Pull request closed without merging**